manifest.json

```diff
@@ -3,7 +3,7 @@
   "registry": "mcr.microsoft.com",
   "variables": {
     "RuntimeReleaseDateStamp": "20200211",
-    "SdkReleaseDateStamp": "20200211",
+    "SdkReleaseDateStamp": "20200316",
```

Comment:
Shouldn't the `3B-*` variable references should be replaced with this variable w/this release?

Reply:
Good catch